src/server/contexts.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable import/prefer-default-export */
+import { Request } from "express";
+import Knex from "knex";
+import createMemoizer from "memoredis";
+
+import { config } from "../config";
+import logger from "../logger";
+import { createLoaders, r } from "./models";
+import { Memoizer } from "./types";
+
+export interface SpokeDbContext {
+  schema: string;
+  master: Knex;
```

Comment:
```suggestion
  primary: Knex;
```
The world seems to be going towards this, we should follow?